tests/integration/security/jwt_test.go

```diff
@@ -325,3 +326,130 @@ func TestAuthnJwt(t *testing.T) {
 			}
 		})
 }
+
+// TestRequestAuthentication tests beta authn policy for jwt.
+func TestRequestAuthentication(t *testing.T) {
+	testIssuer1Token := jwt.TokenIssuer1
+
+	framework.NewTest(t).
+		RequiresEnvironment(environment.Kube).
+		Run(func(ctx framework.TestContext) {
+			ns := namespace.NewOrFail(t, ctx, namespace.Config{
+				Prefix: "req-authn",
+				Inject: true,
+			})
+
+			// Apply the policy.
+			namespaceTmpl := map[string]string{
+				"Namespace": ns.Name(),
+			}
+			jwtPolicies := tmpl.EvaluateAllOrFail(t, namespaceTmpl,
+				file.AsStringOrFail(t, "testdata/requestauthn/simple-jwt-policy.yaml.tmpl"),
+			)
+			g.ApplyConfigOrFail(t, ns, jwtPolicies...)
+			defer g.DeleteConfigOrFail(t, ns, jwtPolicies...)
+
+			var a, b, c, d, e echo.Instance
+			echoboot.NewBuilderOrFail(ctx, ctx).
+				With(&a, util.EchoConfig("a", ns, false, nil, g, p)).
+				With(&b, util.EchoConfig("b", ns, false, nil, g, p)).
+				With(&c, util.EchoConfig("c", ns, false, nil, g, p)).
+				With(&d, util.EchoConfig("d", ns, false, nil, g, p)).
```

Comment:
It seems `d` and `e` is not used in the test.

Reply:
May add some test cases that query `d` to verify they are not affected by policies.

Reply:
add d, remove e.